automation/test.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -xe
+
+make cluster-down
```

Comment:
I don't see where this script is used but I think it should be calling other shell scripts instead of `make cluster-down ...`

Reply:
this could be plugged into the Makefile.  Maybe a new recipe that ties them all together? `make test` or something?

Reply:
The files that will be placed under automation directory, will be executed by STDCI.
I'm not sure we need a recipe for it.